spec/controllers/v0/disability_compensation_forms_controller_spec.rb

```diff
@@ -66,13 +38,7 @@
 
     context 'lighthouse staging' do
       before do
-        Flipper.enable('disability_compensation_lighthouse_brd')
-        Flipper.enable('disability_compensation_staging_lighthouse_brd')
-      end
-
-      after(:all) do
-        Flipper.disable('disability_compensation_lighthouse_brd')
-        Flipper.disable('disability_compensation_staging_lighthouse_brd')
+        ENV['DD_ENV'] = 'eks-staging'
```

Comment:
Why is this needed? I'm concerned that this line (and `ENV['DD_ENV'] = 'eks-prod'` above) could cause problems during parallel test execution. Can you mock this instead?